### PR TITLE
stm32g0: support++

### PR DESCRIPTION
**Pull request by grevaillot**
Hi,

Here's some more g0 support, making it a bit more usefull - adding iwdg/usart/spi/i2c/rng/crc and timer.
Not much new stuff here, a couple of new regs and bits for Timer and RNG but that's all.

I need to review adc and dma/dmamux stuff, but that will wait the next patchset

### CODE REVIEW

**Comment by ChuckM** [comment]
+1

File: lib/stm32/g0/Makefile [hunk 1]
```diff
@@ -39,6 +39,7 @@ OBJS += flash.o flash_common_all.o
 OBJS += gpio_common_all.o gpio_common_f0234.o 
 OBJS += pwr.o
 OBJS += rcc.o rcc_common_all.o
+OBJS += iwdg_common_all.o
```
Comment (karlp) [inline lib/stm32/g0/Makefile]: this is actually alphabetically sorted now, let's try and keep it that way, easier to find things.

**Comment by karlp** [comment]
Looks good, I will sort the .o names when I merge it, no need to repush anything.

**Comment by karlp** [comment]
merged, there's a few things like TIM5 not existing, but those are problems with the timer_common-all common code, not this patch, so ... "out of scope"  thanks for this!

**Pull request merged**